packages/easy-coding-standard/src/Console/Output/JsonOutputFormatter.php

```diff
@@ -88,7 +88,7 @@ private function createBaseErrorsArray(ErrorAndDiffResult $errorAndDiffResult):
     {
         return [
             'meta' => [
-                'version' => $this->configuration->getPrettyVersion(),
+                'version' => $this->configuration->getPrettyVersion() ?: '%s',
```

Comment:
@TomasVotruba I am not sure with this tweak, but in github, it runs test ok.

Reply:
I'll merge it and try look at it locally. It will be some weird stuff

Reply:
Thank you for looking into it :+1: